### ISSUE TITLE
Did you reproduce the full version?

### ISSUE THREAD

**Issue by kongyan66** [WangChyanhassth-2say/BlazePose_torch#1]

**Comment by WangChyanhassth-2say** [comment]
Hi,
If you run the function main(), you can see the size of the model.
The total params size is about 2.66847M, which is between full-version(3.5M) and lite-version(1.3M) given in the paper.

**Comment by kongyan66** [comment]
Thank your reply, I wonder if the training and validation code is implemented ？

**Comment by WangChyanhassth-2say** [comment]
Sorry that I'm on a vacation now, the code is not on hand.   
I've trained this model with its heatmap-branch on coco before, and its AP goes to 65 I remember.  
The training code you may refer [this repo](https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation) I think.

**Comment by kongyan66** [comment]
Thanks a lot, i have a question: Does the regression branch participate in training? How to design the loss?

**Comment by WangChyanhassth-2say** [comment]
By now I only have tried training with the heatmap-branch, with which you can replace the layers after the heatmap output layer with nn.Sequential() -- which means they are useless.   

The regression part is in theory workable despite I don't have much time training it nowadays. You can try training it by yourself.  

To the loss I think MSELoss is just ok, but you may try with [RLE](https://github.com/Jeff-sjtu/res-loglikelihood-regression) cause it seems better.

It's grateful for you to tell me if you succeed.

**Comment by kongyan66** [comment]
ok, i'll try it, thanks again！

**Comment by WangChyanhassth-2say** [comment]
Hi @kongyan66 .  
I've uploaded the training code and the performance now, seeing if they help: )

**Comment by kongyan66** [comment]
Congratulations, i will try it!
